### PR TITLE
Bump flipper* from 0.20.4 to 0.26.0

### PR DESCRIPTION
**Pull request by mkllnk**
#### What? Why?

@filipefurtad0 noticed this warning:

> Flipper::Middleware::Memoizer appears to be running twice. Read how to resolve this at https://github.com/jnunemaker/flipper/pull/523

That warning displayed after upgrading Flipper which was done within the rails7 branch. I noticed that the release notes contained some [upgrade instructions](https://github.com/jnunemaker/flipper/blob/main/Changelog.md#0210) and I simply followed them.

This removes the warnings on the rails7 branch. This pull request replaces:

* #10501

#### What should we test?

- Log in as admin.
- Visit /admin/feature-toggle/features
- The listed features should be the same as before.
- Enable a feature fully and check that it works.
- Enable a feature for one user and check that it works only for that user.
- Delete a feature, try to visit the page that uses that feature and then check that it re-appears in the admin interface.

#### Release notes

Changelog Category: Technical changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

**Review by jibees — APPROVED** [review]
Thanks 🙏

File: app/models/spree/user.rb [hunk 1]
```diff
@@ -148,10 +148,6 @@ def last_incomplete_spree_order
       spree_orders.incomplete.where(created_by_id: id).order('created_at DESC').first
     end
 
-    def flipper_id
```
Comment (abdellani) [inline app/models/spree/user.rb:151]: There is a test related to [flipper_id](https://github.com/openfoodfoundation/openfoodnetwork/blob/5dd27378118ca9626ceac0f71da972971d47a156/spec/models/spree/user_spec.rb#L203)
Do we keep it?

**Review by abdellani — APPROVED** [review]
It looks good. 

I'm just wondering if we still need this [unit test](https://github.com/openfoodfoundation/openfoodnetwork/blob/5dd27378118ca9626ceac0f71da972971d47a156/spec/models/spree/user_spec.rb#L203) 


Thank you.

File: app/models/spree/user.rb (hunk 1 above)
Comment (jibees) [reply app/models/spree/user.rb:151]: I think it's a great idea, since this id is already used in some server, and it's fine to check it did not change (and then we don't need any data migration)

Comment (abdellani) [reply app/models/spree/user.rb:151]: @jibees 
I did a quick search for `flipper_id`, and the only two occurrences were on the spec file.
image
is it used under another name?

Comment (mkllnk) [reply app/models/spree/user.rb:151]: Good find. I agree with both of you. It's good that we kept the spec to check that the flipper id didn't change. But now that we verified that the flipper id is the same as before, we can remove the spec and trust flipper to tell us in release notes if it needs changing. Hm. :thinking: No, I'm changing my mind. Let's keep the spec in case it ever changes and we miss it. It's better to find out in the spec than to have all features for particular users deactivated by accident.

While we don't use the id in the code, we enter it manually in the flipper-ui as admin to activate a feature for a particular user. So we do rely on it. Maybe a better test case would be to enable a feature for an id and then make sure that it's recognised as enabled for a user with that id.

I would also like if we didn't have to enter the id but could select a user in the UI but that would be a new feature within Flipper. I don't think we have capacity for that right now.

To summarize, I think that we should keep the spec as is for now until we find a bug that needs a different spec.

Comment (jibees) [reply app/models/spree/user.rb:151]: > Let's keep the spec in case it ever changes and we miss it. It's better to find out in the spec than to have all features for particular users deactivated by accident.

👍
